### ISSUE TITLE
7.0.5-5 testsuite fails here

### ISSUE THREAD

**Issue by pgajdos** [ImageMagick/ImageMagick#439]
t/filter.t:

testFilterCompare('input.miff', "fuzz=>$fuzz", 'reference/filter/Opaque.miff', 'Opaque', q/color=>"#e23834", fill=>"green"/, 0.0, 0.0);

gives

Set(fuzz=>3276) ...
Opaque(color=>"#e23834", fill=>"green") ...
  mean-error=1.25576800768617e-05, maximum-error=0.282352941176471
not ok 37

**Comment by dlemstra** [comment]
I am unable to reproduce your issue here. Can you add the output from `test-suite.log` to this message?

**Comment by pgajdos** [comment]
Yes:

```
=========================================
   ImageMagick 7.0.5: ./test-suite.log
=========================================

# TOTAL: 86
# PASS:  86
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2
```
Forgot to add that with 7.0.5-4 I do not get the failure on the same system.

**Comment by dlemstra** [comment]
All tests are passing??

**Comment by pgajdos** [comment]
`make check` is successful, yes. But `t/filter.t` comes from PerlMagick's `make test`.

**Comment by mikayla-grace** [comment]
Try https://www.imagemagick.org/download/beta/ImageMagick-7.0.5-6.tar.bz2, the latest beta release.  Does PerlMagick/t/filter.t still fail?  All the tests pass for us on our various unit testing hosts.

**Comment by pgajdos** [comment]
With 7.0.5-6, the filter.t succeeded for me. Consider this invalid/fixed for now, I will retest with 7.0.5-6 final again and will let you know in case of troubles. Thanks.
